res/controllers/Roland_DJ-505-scripts.js

```diff
@@ -1072,9 +1071,9 @@ DJ505.PadSection.prototype.controlToPadMode = function (control) {
     case DJ505.PadMode.VELOCITYSAMPLER:
         mode = this.modes.velocitysampler;
         break;
-    case DJ505.PadMode.LOOP:
-        mode = this.modes.loop;
-        break;
+    //case DJ505.PadMode.LOOP:
```

Comment:
Please add a comment that loop mode might be enabled in the future if Mixxx supports the required features.

Reply:
Done. I also added comments regarding the other missing modes... Sorry, I'm too lazy put them into a separate PR ;-)